.github/workflows/acceptance-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: Acceptance tests
+
+on:
+  pull_request: []
+    # paths: ['renku/service']
```

Comment:
```suggestion
    paths: ['renku/service', 'renku/core','helm-chart']
```

Reply:
Thanks, I haven't finalized this yet since I'm still testing so I'll keep this here as a reminder